docs/src/pages/components/data-grid/editing/EditRowModelControlGrid.js

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react';
+import { DataGrid } from '@material-ui/data-grid';
+import {
+  randomCreatedDate,
+  randomTraderName,
+  randomUpdatedDate,
+} from '@material-ui/x-grid-data-generator';
+
+const columns = [
+  { field: 'name', headerName: 'Name', width: 180, editable: true },
+  { field: 'age', headerName: 'Age', type: 'number', editable: true },
+  {
+    field: 'dateCreated',
+    headerName: 'Date Created',
+    type: 'date',
+    width: 180,
+    editable: true,
+  },
+  {
+    field: 'lastLogin',
+    headerName: 'Last Login',
+    type: 'dateTime',
+    width: 220,
+    editable: true,
+  },
+];
+
+const rows = [
+  {
+    id: 1,
+    name: randomTraderName(),
+    age: 25,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 2,
+    name: randomTraderName(),
+    age: 36,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 3,
+    name: randomTraderName(),
+    age: 19,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 4,
+    name: randomTraderName(),
+    age: 28,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 5,
+    name: randomTraderName(),
+    age: 23,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+];
+
+export default function EditRowModelControlGrid() {
+  const [editRowsModel, setEditRowsModel] = React.useState({});
+
+  const handleEditRowModelChange = React.useCallback((params) => {
+    setEditRowsModel(params.model);
+  }, []);
+
+  return (
+    <div style={{ width: '100%' }}>
+      <DataGrid
+        rows={rows}
+        columns={columns}
+        editRowsModel={editRowsModel}
+        onEditRowModelChange={handleEditRowModelChange}
+        autoHeight
+      />
+      <code>{JSON.stringify(editRowsModel)}</code>
```

Comment:
What do you think about rendering above (so it's more obvious) and with a label. My first thought when I saw:

<img width="311" alt="Screenshot 2021-04-09 at 13 15 17" src="https://user-images.githubusercontent.com/3165635/114172234-af66d300-9935-11eb-9ec3-e170549777cb.png">

what: "oh, a typo in the code, it's broken".

```suggestion
      editRowsModel state: <code>{JSON.stringify(editRowsModel)}</code>
      <DataGrid
        rows={rows}
        columns={columns}
        editRowsModel={editRowsModel}
        onEditRowModelChange={handleEditRowModelChange}
        autoHeight
      />
```